skimage/data/_binary_blobs.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+from ..filters import gaussian_filter
+
+
+def binary_blobs(length=512, blob_size_fraction=0.1, n_dim=2,
+                 volume_fraction=0.5, seed=None):
+    """
+    Generate synthetic binary image with several blob-like rounded objects.
+
+    Parameters
+    ----------
+    length : int, default 512
+        Linear size of output image.
+    blob_size_fraction : float, default 0.1
+        Typical linear size of blob, as a fraction of ``length``, should be
+        smaller than 1.
+    n_dim : int, default 2
+        Number of dimensions of output image.
+    volume_fraction : float, default 0.5
+        Fraction of image pixels covered by the blobs (where the output is 1).
+        Should be in [0, 1].
+    seed : int, default 0
+        Seed to initialize the random number generator.
+
+    Returns
+    -------
+    blobs : ndarray of bools
+        Output binary image
+
+    Examples
+    --------
+    >>> blobs = binary_blobs(length=256, blob_size_fraction=0.1)
+    >>> # Finer structures
+    >>> blobs = binary_blobs(length=256, blob_size_fraction=0.05)
+    >>> # Blobs cover a smaller volume fraction of the image
+    >>> blobs = binary_blobs(length=256, volume_fraction=0.3)
+    """
+    if seed is None:
```

Comment:
Oh, don't fix it, pass it only when it is explicitly given. See `scikit-learn`'s `check_random_state` function, or my copy, [`normalise_random_state`](https://github.com/microscopium/microscopium/blob/master/microscopium/_util.py#L31). (I like my name better. =P)

Anyway, only fix the seed when given, otherwise give non-reproducible behaviour, that's kind of the point!